jenkins/jobs/macros-common.yaml

```diff
@@ -108,6 +118,8 @@
           sudo chmod -R 0777 ~/.ccache/
           #sudo chmod -R 0777 /home/jenkins/.ccache/
           pip install -r swift-uploader/requirements.txt
+          mkdir -pm 777 logs/domzero_logs
```

Comment:
I see, sorry that I don't know which is the correct mode, just though 755 is enough, so ignore my comment here